examples/pymc3_howto/profiling.ipynb

```diff
@@ -31,16 +31,6 @@
     "print(f\"Running on PyMC3 v{pm.__version__}\")"
```

Comment:
<p>I realized that data loading was not covered in the style guide, so I added a bullet point for that. Ideally they should use a try except clause: https://github.com/pymc-devs/pymc3/wiki/PyMC3-Jupyter-Notebook-Style-Guide</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/143/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>